server/controllers/landlordController.js

```diff
@@ -3,22 +3,6 @@ const queries = require('../models/queries');
 
 const landlordController = {};
 
-//add landlord controller - check if req body isLandlord, if yes, post landlord; 
-landlordController.postLandlord = async (req, res, next) => {
```

Comment:
Need to discuss backend changes. changes to `landlord` in db should be segregated from changes to `user` in db

Reply:
put into landlord controller